### PR TITLE
Connection unit tests

### PR DESCRIPTION
**Pull request by vogdb**
Based on #27

### CODE REVIEW

File: test/helpers/websocket.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+require('mock-socket');
```
Comment (tomaszdurka) [inline test/helpers/websocket.js]: Isn't this missing some declaration?

Comment (vogdb) [inline test/helpers/websocket.js]: Unfortunately no. Specifics of this library :( It imports to global.

**Comment by vogdb** [comment]
@tomaszdurka please review

**Comment by tomaszdurka** [comment]
So this bascically tests that messages are forwaded within connection. Looks good for me.
We still don't test things like queuing.
Let's merge this one though.

**Pull request merged**